src/app/components/OpptsCard/index.jsx

```diff
@@ -0,0 +1,17 @@
+import opptsData from '../../assets/opptsData.json'
+import styles from './opptsCard.module.css'
+
+const OpptsCard = () => {
+  return (
+    <div className={styles.opptsContainer}>
+      {opptsData.map((oppt, index) => (
+        <div className={styles.opptItem}>
```

Comment:
Såg här att du råkat glömma att lägga in en key prop för att kunna mappa igenom index.

Mitt förslag:         
<div key={index} className={styles.opptItem}>

Reply:
Glöm denna kommentar! Det funkade nu av någon anledning. Det är approved ✅